File: sqlLed.php
<!DOCTYPE html>
<html>

<head>
    <title>
        GPIO LED
    </title>
</head>

<body style="text-align:center;">

    <h1 style="color:green;">
        GPIO LED Control
    </h1>

    <?php
        //Use an if statement to trigger php functions
        if (array_key_exists('offButton', $_POST)) {  //listen for button press
            offButton(); 
        } 
        else if (array_key_exists('onButton', $_POST)) { 
            onButton(); 
        } 
        
        //Define our php functions
        function offButton() { //turns off led
            $servername = "localhost"; //use localhost because file is already on the server
            $username = "root";
            $password = "guzzo";
            $dbname = "gpioControl"; //database name

            // Create connection
            $conn = new mysqli($servername, $username, $password, $dbname);
            // Check connection
            if ($conn->connect_error) {
                die("Connection failed: " . $conn->connect_error);
            }

            $sql = "UPDATE led SET toggleOne=1"; //update value to turn off the led
            $result = $conn->query($sql); //holds result of the query

            //Tell user if sql query failed
            if ($conn->query($sql) === TRUE) {
                echo "LED off". "<br>";
            }
            else {
                echo "Error: " .$sql . "<br>" . $conn->error;
            }

            //Close connection
            $conn->close(); 
        }
            
        function onButton() { //turns on led
            $servername = "localhost"; //use localhost because file is already on the server
            $username = "root";
            $password = "guzzo";
            $dbname = "gpioControl"; //database name

            // Create connection
            $conn = new mysqli($servername, $username, $password, $dbname);
            // Check connection
            if ($conn->connect_error) {
                die("Connection failed: " . $conn->connect_error);
            }

            $sql = "UPDATE led SET toggleOne=2 "; //update value to turn on the led
            $result = $conn->query($sql); //holds result of the query

            //Tell user if sql query failed
            if ($conn->query($sql) === TRUE) {
                echo "LED on". "<br>";
            }
            else {
                echo "Error: " .$sql . "<br>" . $conn->error;
            }

            //Close connection
            $conn->close(); 
        }  
    ?>

    <!-- Use a form to automatically execute our php functions -->
    <form method="post">
        <input type="submit" value="Off" class="button" name="offButton" />

        <input type="submit" value="On" class="button" name="onButton" />
    </form>

</body>

</html>

File: sqlLED.py
# Imports
import pymysql
import RPi.GPIO as GPIO
import time

# GPIO settings
GPIO.setmode(GPIO.BCM) 
GPIO.setwarnings(False) 
GPIO.setup(21,GPIO.OUT) 

# Store string for comparison later
offdata = "((1,),)"

# Create a loop that will constantly check the database
print("Loop starting...") 
while 0 == 0:
	# Open database connection
    db = pymysql.connect(host="localhost", user="root", passwd="guzzo", db="gpioControl")

	# Create a cursor object using cursor() method
    cur = db.cursor()

	# Select desired value from table
    sql = "SELECT toggleOne FROM led"

    # Execute sql statement
    cur.execute(sql)
    
    # Retrieve data from the sql statement
    rows = cur.fetchall() 
    
    # Show us our sql statement
    print(rows)
    
    r = str(rows) # Stringify sql statement to compare with our offdata string
    
    # Compare output data to turn on and off light
    if r == offdata: #check our data to turn off light
        print("LED off") 
        GPIO.output(21,GPIO.LOW) #turn off led
        time.sleep(1) #add a delay to not DDOS the database
    else:
        print("LED on") 
        GPIO.output(21,GPIO.HIGH) #turn on led
        time.sleep(1) 

	# Disconnect from server
    cur.close()
    db.close()
